test/42-device-api-v2.spec.ts

```diff
@@ -289,12 +289,12 @@ describe('SupervisorAPI [V2 Endpoints]', () => {
 							.body,
 					);
 				});
-			// Deactivate localmode
```

Comment:
Removed this because it was interfering with the splash tests, since the config listener on the firewall was being triggered 🙄

Reply:
In the short term, let's stub `.on` / `.once` listeners in another PR?

Reply:
Yeah, I think we should try that on another PR and fix whatever breaks.